src/map/pet.c

```diff
@@ -1254,23 +1254,23 @@ static void pet_read_db(void)
 		DBPATH"pet_db.conf",
 		"pet_db2.conf"
 	};
-	int i;
+	int i, count = 0;
```

Comment:
while you changing this line, ``int i``, better put inside for as ``for (int i = 0 ...``, before it was impossible due outdated visual studio version.